src/settings.py

```diff
@@ -246,6 +249,17 @@ def _apply(
                 {"sudoOption": ["!authenticate"] if passwordless_sudo else []},
             )
 
+        global_portal_settings_path = Path(f"{PORTAL_SETTINGS_DIR}/global_settings.json")
+        portal_allow_edit_email = form.get("portal_allow_edit_email", self.get("security.portal.portal_allow_edit_email"))
+        portal_allow_edit_email_alias = form.get("portal_allow_edit_email_alias", self.get("security.portal.portal_allow_edit_email_alias"))
+        portal_allow_edit_email_forward = form.get("portal_allow_edit_email_forward", self.get("security.portal.portal_allow_edit_email_forward"))
+        write_to_json(
+            str(global_portal_settings_path), {"allow_edit_email": portal_allow_edit_email,
+                                               "allow_edit_email_alias": portal_allow_edit_email_alias,
+                                               "allow_edit_email_forward":portal_allow_edit_email_forward
+                                               }, sort_keys=True, indent=4
+        )
```

Comment:
I think this should already managed by ConfigPanel. It doesn't work without ?

Reply:
The issue currently is that we store 2 config at different places. One global setting which is not accessible by the portal API and an other one more specific to the portal api which is accessible by the portal api (and which is specific to the domain).

The issue currently is that to me this setting is expected to be global it don't really make sens to be different between 2 domain as it's also a security thing. So I added this parameters in the global setting. But this can't be read by the config panel. So it's why I added one global config for the portal API (sychromized from the global setting) so the portal api can read the config.